curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md

```diff
@@ -7,11 +7,14 @@ forumTopicId: 301540
 
 ## Description
 <section id='description'>
-Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your mLab database URI in the private <code>.env</code> file as MONGO_URI. Connect to the database using <code>mongoose.connect(&lt;Your URI&gt;)</code>
+Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your mLab database URI in the private <code>.env</code> file as MONGO_URI. Connect to the database using the following syntax:
+
+```js
+mongoose.connect(<Your URI>, { useNewUrlParser: true, useUnifiedTopology: true }); 
```

Comment:
```suggestion
mongoose.connect(process.env.MONGO_URI, { useNewUrlParser: true, useUnifiedTopology: true }); 
```
What do you think?  I ended up googling 'glitch environment variables' to figure this out, so perhaps a little hand-holding would not hurt.

Reply:
I think this just turns it into a copy/paste vs. understanding to use process.env.  They have already learned about process .env before in the other section, so I think we should leave this part alone.

Reply:
Oh, I see.  I'd forgotten it had been introduced before.  Since that's the case, sure, let's leave it.

Reply:
I'm fine with this change, but what about [the boilerplate](https://github.com/freeCodeCamp/boilerplate-mongomongoose/blob/gomix/myApp.js#L9) - this is why the instructions should be removed from the boilerplates.

Reply:
@moT01 Don't you already have PR or issue for the boilerplates?

Reply:
yes, [here](https://github.com/freeCodeCamp/freeCodeCamp/issues/35541) - but it doesn't have much discussion. Maybe for this one, it would be easiest to just update the boilerplate and glitch project, and deal with the other issue separately.